README.md

```diff
@@ -1,5 +1,40 @@
-# Node-RED Quantum
-Quantum computing functionality for Node-RED.
+# node-red-quantum
```

Comment:
```suggestion
# Node-RED Quantum
```
Personally I prefer title-case here, but if people prefer this way we can stick with it.

Reply:
I'm afraid it's a Node-RED convention and we're not allowed to change it: https://flows.nodered.org/search?type=node .